pkg/k8shandler/rbac.go

```diff
@@ -92,19 +92,32 @@ func CreateOrUpdateRBAC(dpl *api.Elasticsearch) error {
 		return err
 	}
 
-	subject = newSubject(
-		"ServiceAccount",
-		dpl.Name,
-		dpl.Namespace,
-	)
-	subject.APIGroup = ""
+	esList := &api.ElasticsearchList{
```

Comment:
can we add a small comment here so in the future we can remember what and why we're doing this?

Reply:
done